src/particle/particle/particle.py

```diff
@@ -1015,6 +1015,76 @@ def from_evtgen_name(cls: type[Self], name: str) -> Self:
         """
         return cls.from_pdgid(EvtGenName2PDGIDBiMap[name])
 
+    @classmethod
+    def from_nucleus_info(
+        cls: type[Self],
+        z: int,
+        a: int,
+        anti: bool = False,
+        i: int = 0,
+        l_strange: int = 0,
+    ) -> Self:
+        """
+        Get a nucleus particle from the neutron and proton count.
+        As described here https://pdg.lbl.gov/2007/reviews/montecarlorpp.pdf:
```

Comment:
Better not refer to such an old document. If you really want to say something you can say the name of the document, hence the PDG numbering scheme. Else refer for example to the function at https://github.com/scikit-hep/particle/blob/master/src/particle/pdgid/functions.py#L232 that says what a nucleus PDG ID is. No need to then repeat it here.